### ISSUE TITLE
“0 attributes created. X attributes could not be saved" error when importing email addresses as 'target-email' attribute type

### ISSUE THREAD

**Issue by bkime** [MISP/MISP#3931]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | RedHat 7.5
| OS version (client)       | Windows 10 Enterprise 
| PHP version               | 5.6.25
| MISP version / git hash   | v2.4.97 (ce3c78cd7db60812d0147ced992a7650509d31da)
| Browser                   | Google Chrome Browser Version 71.0.3578.80 (Official Build) (64-bit)

### Expected behavior
When adding email addresses to an event via "Populate from..." -> "Freetext Import", we can successfully change from the default email-dst attribute type to target-email type before completing the import.

### Actual behavior
When attempting to add attributes to an event using the “Populate from…” -> “Freetext Import” feature, the error “0 attributes created. X attributes could not be saved. This may be due to attributes with similar values already existing.” is displayed only when selecting the category “Targeting Data” and type “target-email”. The attributes are not imported via this method (not just an error message). However, you can import the same data to the same event, category, and type one at a time not via the “Populate from” feature. You can also use the “Populate from” feature for bulk imports of the same data if another category and/or type is selected.

### Steps to reproduce the behavior
1. Create a new event, fill out information, click 'add'
2. Select "Populate from..." and then "Freetext Import"
3. Paste email addresses in freetext import tool, click 'submit'
4. On results page, change all from email-dst to target-email and click 'Submit attributes'.
5. Get error message at the top: 0 attributes created. X attributes could not be saved. Reason for the failure: {"type":["Options depend on the selected category."]}

**Comment by bkime** [comment]
Team, has this been addressed? I was able to reproduce this bug in version .101.

**Comment by adulau** [comment]
Indeed, I can reproduce it in .102. So the issue is the following, `target-email` is in `Targeting data` category as it describes the real target. The two categories proposed are different and you cannot have 'target-email' in other category. We will have a look at the bug.
